blocks/hero/hero.css

```diff
@@ -18,8 +18,8 @@
   font-weight: 900;
   line-height: 1;
   color: #000;
-  margin: 8px 0 10px;
-  padding: 25px 0 10px;
+  margin: 0 10px;
```

Comment:
@veejayricana looks like this broke the margin for category pages and search page hero in mobile view

<img width="196" alt="image" src="https://github.com/hlxsites/accenture-newsroom/assets/1553152/be33e500-ff37-4b74-a8dd-691be560f18f">

<img width="197" alt="image" src="https://github.com/hlxsites/accenture-newsroom/assets/1553152/95974052-6493-4282-8b7e-e75c812d4585">

Can you please check?

Reply:
Hi @ravkiran, I will create new branch for this one to fix this issue in smaller viewport like mobile.

Reply:
I already commit changes for this @ravkiran